tendrl_node_agent/monitoring/plugins/ifspeed.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Dont remember how this ifspeed.sh executed. Can you explain in comments in the start of source?